### PR TITLE
Add id field to resources bigtable and bigquery

### PR DESCRIPTION
**Pull request by venkykuberan**
**Release Note Template for Downstream PRs (will be copied)**

```release-note:REPLACEME

```

### CODE REVIEW

File: third_party/terraform/website/docs/r/bigtable_instance.html.markdown [hunk 1]
```diff
@@ -86,7 +86,9 @@ for a `DEVELOPMENT` instance.
 
 ## Attributes Reference
 
-Only the arguments listed above are exposed as attributes.
+* id - an identifier for the resource with format projects/{{project}}/instances/{{name}}
+
+* name - The unique name of the requested big table instance. Values are of the form projects/<project>/instances/<instanceId>.
```
Comment (rileykarson) [inline third_party/terraform/website/docs/r/bigtable_instance.html.markdown]: This isn't an exported field on this resource. It's a user-configured field. https://github.com/GoogleCloudPlatform/magic-modules/pull/3207/files#diff-cabec47f1c3393e21fdded661c4b35a8R51

Unlike the other resource we were talking about, we use a field called `name` to supply what the API calls the instanceId.

File: third_party/terraform/website/docs/r/bigtable_table.html.markdown [hunk 2]
```diff
@@ -58,6 +58,10 @@ The following arguments are supported:
 
 ## Attributes Reference
 
+* id - an identifier for the resource with format projects/{{project}}/instances/{{instance}}/tables/{{name}}
+
+* name - an identifier for the resource with format projects/{{project}}/instances/{{instance}}/tables/{{tableId}}
```
Comment (rileykarson) [inline third_party/terraform/website/docs/r/bigtable_table.html.markdown]: Same as above.

File: third_party/terraform/website/docs/r/bigtable_gc_policy.html.markdown [hunk 3]
```diff
@@ -29,16 +29,16 @@ resource "google_bigtable_instance" "instance" {
 
 resource "google_bigtable_table" "table" {
   name          = "tf-table"
-  instance_name = google_bigtable_instance.instance.name
```
Comment (rileykarson) [inline third_party/terraform/website/docs/r/bigtable_gc_policy.html.markdown]: Have you tested this resource? I"m not confident that the generated test will succeed.

Comment (venkykuberan) [reply third_party/terraform/website/docs/r/bigtable_gc_policy.html.markdown]: Yes tested locally, it works only for `name` attribute at this point. To support for `id` attribute an issue has been raised. Once that's implemented i will pick up this PR

File: third_party/terraform/website/docs/r/bigtable_instance.html.markdown (hunk 1 above)
Comment (venkykuberan) [reply third_party/terraform/website/docs/r/bigtable_instance.html.markdown]: removed the field

File: third_party/terraform/website/docs/r/bigtable_table.html.markdown (hunk 2 above)
Comment (venkykuberan) [reply third_party/terraform/website/docs/r/bigtable_table.html.markdown]: removed the field

**Pull request closed without merging**